### ISSUE TITLE
Docs: Format document

### ISSUE THREAD

**Issue by yuluo-yx** [LinuxSuRen/open-source-best-practice#155]
now, in https://linuxsuren.github.io/open-source-best-practice/.

There are some problems with mixing Chinese and English and formatting problems

e.g.: 
<img width="445" alt="image" src="https://github.com/LinuxSuRen/open-source-best-practice/assets/77964041/890f17a6-0d4d-40fb-8e6d-f88625a43529">
<img width="237" alt="image" src="https://github.com/LinuxSuRen/open-source-best-practice/assets/77964041/9c9c4061-47df-47d5-9dcd-4c86ec325660">

> There are some punctuation issues:
<img width="748" alt="image" src="https://github.com/LinuxSuRen/open-source-best-practice/assets/77964041/521e3301-efd9-4356-bc2d-e26eccdbb889">

**Comment by LinuxSuRen** [comment]
hi @yuluo-yx , thanks for reporting this issue. Please feel free to help to fix it if you are interested in it.

**Comment by yuluo-yx** [comment]
> hi @yuluo-yx , thanks for reporting this issue. Please feel free to help to fix it if you are interested in it.

It's a great honor, I will submit a PR soon to fix it, please assign to me @LinuxSuRen

**Comment by yuluo-yx** [comment]
I have an idea. In open source best practice projects, most of the content is displayed in text. We can establish a set of specifications for writing documents to provide reference when writing documents to avoid these problems.

**Comment by LinuxSuRen** [comment]
> I have an idea. In open source best practice projects, most of the content is displayed in text. We can establish a set of specifications for writing documents to provide reference when writing documents to avoid these problems.

Of course, it could be wonderful if you could take the initiative to make a start. It is crucial to encourage greater collaboration among all stakeholders in this project.

I have a new idea. We could also convert the Markdown files into a PDF. And publish it in GitHub release.

**Comment by yuluo-yx** [comment]
Very happy to contribute to this. 💪

**Comment by yuluo-yx** [comment]
> I have a new idea. We could also convert the Markdown files into a PDF. And publish it in GitHub release.

A good idea, we can write a roadmap document for this to record some of our ideas and implement them later. 😄

**Comment by yuluo-yx** [comment]
![image](https://github.com/LinuxSuRen/open-source-best-practice/assets/77964041/7b95e63a-89b3-45c2-9ef7-b3d3cc3bd900)

![image](https://github.com/LinuxSuRen/open-source-best-practice/assets/77964041/232ccdb4-a851-4878-a900-6ff761e89c71)

In this case, it should be a problem with the page display. The file format is normal when viewed in the editor.
